tck/TCK_NOTES.adoc

```diff
@@ -0,0 +1,113 @@
+= Named Graphs =
+
+The TCK relies on a number of included named graphs.
+
+
+== Metadata File ==
+
+Each named graph is declared using a JSON file that references various Cypher script files for importing the graph as well as includes some additional statistical information.
+
+This JSON file is called the metadata file.
+
+=== Statistics Generation ===
+
+The following sections contain queries for the original Neo4j implementation of Cypher that have been used to generate the statistical information for some of the named graphs used in the TCK.
+
+These queries use the APOC procedure library for the generation of JSON results.
```

Comment:
I would suggest linking to the APOC library, so that readers are not left mystified